java/avro-converter/src/main/java/com/microsoft/AvroConverter.java

```diff
@@ -0,0 +1,133 @@
+package com.microsoft;
```

Comment:
This package name looks wrong. should be more specific to the package name we intend to release I think?

For reference our Avro serializer is com.microsoft.azure.schemaregistry.kafka.avro so this one should be more like com.microsoft.azure.schemaregistry.kafka.connect.avro